pages/linux/lsscsi.md

```diff
@@ -0,0 +1,15 @@
+# lsscsi
+
+> List SCSI devices (or hosts) and their attributes.
+
+- List all SCSI devices:
+
+`lsscsi`
+
+- List all SCSI devices with detailed attributes:
+
+`lsscsi -L`
+
+- List all SCSI devices with human readable disk capacity:
+
+`lsscsi -s`
```

Comment:
I don't see the `-s` option here - https://linux.die.net/man/8/lsscsi. Are you using linux or macOS ?

Reply:
Yes. I use Ubuntu actually.
We can find it use `man lsscsi` on Linux:
```
    -s, --size
              Print disk capacity in human readable form.
```